tools/accuracy_checker/openvino/tools/accuracy_checker/annotation_converters/background_matting.py

```diff
@@ -130,3 +130,76 @@ def convert(self, check_content=False, progress_callback=None, progress_interval
         return ConverterReturn(
             annotations, self.get_meta(), content_errors
         )
+
+
+class BackgroundMattingSequential(BackgroundMattingConverter):
```

Comment:
base class has no parameters `background_prefix`, `with_background`, `with_alpha`. please add them and possibly `background_postfix` and `backgrounds_dir` in this class

Reply:
This is my mistake, forgot to add them to the `BackgroundMattingSequential` class. Probably, these new parameters must be in `BackgroundMattingSequential`, not in the base class?

Reply:
Yes, that's what I meant. My mistake, I didn't explain correctly

Reply:
Fixed.